src/connect/QRCodeConnectScreen.tsx

```diff
@@ -66,10 +68,29 @@ const QRCodeConnectScreen = ({
   warning,
   connect,
 }: QRCodeConnectScreenProps) => {
+  
+  const [darkMode, setDarkMode] = useState(false);
+
+  useEffect(() => {
+
+    setTheme();
+    
+    //Set QR Code Theme
+    //Warning: QR Code is currently not responsive to theme changes after initial render
+    if(getTheme() == "dark") {
+      setDarkMode(true);
+    }else if (getTheme() == "light") {
+      setDarkMode(false);
+    }else if (getTheme() == "auto"){
+      setDarkMode(window.matchMedia('(prefers-color-scheme: dark)').matches);
+    }
+  }, []);
+
   return (
-    <>
+    <main 
+    className="main">
       <h1
```

Comment:
If I remember correctly, I wrapped this in a fragment to preserve the formatting from the parent `flex-col`. 

Can we put this `className="main"` on the root node?